spec/migrations/20180316164826_update_switch_types_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_migration
+
+describe UpdateSwitchTypes do
+  let(:switch_stub)      { migration_stub(:Switch) }
+  let(:host_stub)        { migration_stub(:Host) }
+
+  migration_context :up do
+    it "migrates a series of representative rows" do
```

Comment:
Do you have any tests for the non-nil switch type case?

Reply:
:+1: added one [here](https://github.com/ManageIQ/manageiq-schema/pull/180/files#diff-a25504ed68df4c44e21f168b8a1671fcR37)